src/client/lobby/lobby.spec.js

```diff
@@ -10,7 +10,6 @@ describe('<Lobby />', () => {
 
```

Comment:
window.cloak isn't required in this file any more, because it's been moved to lobby-page instead?